Packs/ShiftManagement/Scripts/GetUsersOnCall/GetUsersOnCall.py

```diff
@@ -1,16 +1,49 @@
 import demistomock as demisto
 from CommonServerPython import *
 from CommonServerUserPython import *
+import ast
 
 from typing import List
 
 
+def filter_OOO_users(get_users_response):
+    """
+    Given the response with all OnCall users, remove the users that are Out Of Office, using the list `OOO List`.
+    """
+    all_users = get_users_response.get('Contents')
+    if not all_users:
+        return 'No data returned'
+
+    OOO_users_list = demisto.executeCommand('getList', {'listName': 'OOO List'})
+    if is_error(OOO_users_list):
```

Comment:
are we getting an error if the list dose not exist?
if so, I like to split it.
if list dose not exist, log debug that it dose not exist.
for all others, log error.